tests/CMakeLists.txt

```diff
@@ -43,6 +43,17 @@ target_link_libraries(asio_service_stream_test
                       ${BUILD_DIR}/${LIBRARY_OUTPUT_NAME}
                       ${LIBRARIES})
 
+add_executable(asio_stream_test
+               unit/asio_stream_test.cxx
```

Comment:
By the way, in the next PR (for testing), please rename it to `stream_functional_test`.